packages/modern-plugin-rspress/rslib.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from '@rslib/core';
+
+export default defineConfig({
+  lib: [
+    {
+      dts: {
+        bundle: true,
+      },
+      format: 'cjs',
+      syntax: 'es2020',
+      output: {
+        sourceMap: { js: 'source-map' },
```

Comment:
I think we do not need to ship source map for Rspress packages. This can reduce install size.

@Timeless0911 What is your opinion

Reply:
Agree, but why we add sourcemap before?

Reply:
All sourcemap removed.